features/admin.applications.v1/components/edit-application.tsx

```diff
@@ -1080,160 +529,760 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
         </>
     );
 
-    const GeneralApplicationSettingsTabPane = (): ReactElement => (
-        <ResourceTab.Pane controlledSegmentation>
-            <GeneralApplicationSettings
-                accessUrl={ application?.accessUrl }
-                appId={ application?.id }
-                description={ application?.description }
-                discoverability={ application?.advancedConfigurations?.discoverableByEndUsers }
-                imageUrl={ application?.imageUrl }
-                name={ application?.name }
-                application = { application }
-                isLoading={ isLoading }
-                onDelete={ onDelete }
-                onUpdate={ handleApplicationUpdate }
-                featureConfig={ featureConfig }
-                template={ template }
-                isBrandingSectionHidden={ brandingDisabledFeatures.includes(BrandingPreferencesConstants.
-                    APP_WISE_BRANDING_FEATURE_TAG) }
-                readOnly={ readOnly || applicationConfig.editApplication.getTabPanelReadOnlyStatus(
-                    "APPLICATION_EDIT_GENERAL_SETTINGS", application) }
-                data-componentid={ `${ componentId }-general-settings` }
-                isManagementApp={ application.isManagementApp }
-            />
-        </ResourceTab.Pane>
-    );
+    const GeneralApplicationSettingsTabPane = (): ReactElement => (
+        <ResourceTab.Pane controlledSegmentation>
+            <GeneralApplicationSettings
+                accessUrl={ application?.accessUrl }
+                appId={ application?.id }
+                description={ application?.description }
+                discoverability={ application?.advancedConfigurations?.discoverableByEndUsers }
+                imageUrl={ application?.imageUrl }
+                name={ application?.name }
+                application = { application }
```

Comment:
Here we are passing the entire application object, yet we have dozen of props just to pass the properties from the same application object.

Wouldn't it be better to remove the redundant props and use values from the application prop itself?

Reply:
This is an existing code block. https://github.com/wso2/identity-apps/blob/05605b9367987c95ce06add63ce0e95494a90b2d/features/admin.applications.v1/components/edit-application.tsx#L1089-L1093